modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -57,6 +68,8 @@ const RichText = ({
 	visible,
 	...otherProps
 }) => {
+	const {portletNamespace} = useConfig();
```

Comment:
Hey @larissacribeiro, as these constants have no order of precedence, can we order them alphabetically? So:

```
const contents = useMemo(
		() => (editable ? predefinedValue : value ?? predefinedValue),
		[editable, predefinedValue, value]
);
	
const editorRef = useRef();
		
const {portletNamespace} = useConfig();
```

Can we check if SF accepts this ordering?

Reply:
Hi @igor-franca, sure, I will push this now!